examples/jaeger-mixin/alerts.libsonnet

```diff
@@ -0,0 +1,140 @@
+              {{ $labels.job }} {{ $labels.instance }} is experiencing {{ printf "%.2f" $value }}% RPC HTTP errors.
+            |||,
+          },
+        }, {
+          alert: 'JaegerClientSpansDropped',
+          expr: percentErrs('jaeger_reporter_spans', 'result=~"dropped|err"') + '> 1',
+          'for': '15m',
+          labels: {
+            severity: 'warning',
+          },
+          annotations: {
+            message: |||
+              service {{ $labels.job }} {{ $labels.instance }} is dropping {{ printf "%.2f" $value }}% spans.
+            |||,
+          },
+        }, {
+          alert: 'JaegerAgentSpansDropped',
+          expr: percentErrsWithTotal('jaeger_agent_reporter_batches_failures_total', 'jaeger_agent_reporter_batches_submitted_total') + '> 1',
+          'for': '15m',
+          labels: {
+            severity: 'warning',
+          },
+          annotations: {
+            message: |||
+              agent {{ $labels.job }} {{ $labels.instance }} is dropping {{ printf "%.2f" $value }}% spans.
+            |||,
+          },
+        }, {
+          alert: 'JaegerCollectorDroppingSpans',
+          expr: percentErrsWithTotal('jaeger_collector_spans_dropped_total', 'jaeger_collector_spans_received_total') + '> 1',
+          'for': '15m',
+          labels: {
+            severity: 'warning',
+          },
+          annotations: {
+            message: |||
+              collector {{ $labels.job }} {{ $labels.instance }} is dropping {{ printf "%.2f" $value }}% spans.
+            |||,
+          },
+        }, {
+          alert: 'JaegerSamplingUpdateFailing',
+          expr: percentErrs('jaeger_sampler_queries', 'result="err"') + '> 1',
+          'for': '15m',
+          labels: {
+            severity: 'warning',
+          },
+          annotations: {
+            message: |||
+              {{ $labels.job }} {{ $labels.instance }} is failing {{ printf "%.2f" $value }}% in updating sampling policies.
+            |||,
+          },
+        }, {
+          alert: 'JaegerThrottlingUpdateFailing',
+          expr: percentErrs('jaeger_throttler_updates', 'result="err"') + '> 1',
+          'for': '15m',
+          labels: {
+            severity: 'warning',
+          },
+          annotations: {
+            message: |||
+              {{ $labels.job }} {{ $labels.instance }} is failing {{ printf "%.2f" $value }}% in updating throttling policies.
+            |||,
+          },
+        }, {
+          alert: 'JaegerQueryReqsFailing',
+          expr: percentErrs('jaeger_query_requests_total', 'result="err"') + '> 1',
+          'for': '15m',
+          labels: {
+            severity: 'warning',
+          },
+          annotations: {
+            message: |||
+              {{ $labels.job }} {{ $labels.instance }} is seeing {{ printf "%.2f" $value }}% query errors on {{ $labels.operation }}.
+            |||,
+          },
+        }, {
+          alert: 'JaegerCassandraWritesFailing',
```

Comment:
Should we add Elasticsearch alerts as well? This metric wouldn't exist on an instance where only Elasticsearch is used: would that trigger an alert itself, or would it just ignore?